brave/brave-core
# Add Android libs licenses

Resolves https://github.com/brave/brave-browser/issues/29106



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open `brave://credits`.
2. Look for `billing`.
3. Click on the license link to see that the *Android SDK license* is included.
4. Click on the homepage and check that it goes to the *Google Maven Repository*.
